src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -143,7 +150,55 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
     if (onTabSelected) {
       onTabSelected(index);
     }
-    setCurrentIndex(index);
+    setCurrentActiveIndex(index);
+  };
+
+  const onKeyDown = (event: React.KeyboardEvent<HTMLUListElement>) => {
+    // alt + left/right is used by browser / assistive technologies
+    if (event.altKey) {
```

Comment:
oh, this we might need to all the components then? need to research...

Reply:
I think DatePicker/TimePicker is the only one where we are handling left/right keys.

Other than that, Menu is the only one where we are handling up/down keys.

Reply:
This thread may be relevant: https://github.com/GoogleChromeLabs/howto-components/issues/19